docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,201 @@
+# Styling UI components with XAML
+Styling of components in NUI is possible in two ways:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
```

Comment:
@nawab-samsung As I [commented](https://github.com/Samsung/tizen-docs/pull/1386#discussion_r560629916), the meaning of the original sentence was slightly changed during the review.. Your suggestion to this sentence changed the meaning a little bit, so I added "With XAML file, " to keep the original meaning and your update both.

Reply:
@mijin85cho I just broke the sentence, I don't see meaning changing in any way. May be can you please help me understand this. 
Ist part: 
XAML file:

As I get it we are writing a description for the XAML file that has a colon at the end, which implies that whatever is after the colon is related to the XAML file. 
2nd part:  designed for advanced themes.

3rd part: continuation of description. 

The styles change in runtime and can be shared between the applications. 


Where have I changed the meaning? I would love to learn from you. You like it that way is fine. but when we already have  XAML file at the start, why we need redundancy in sentences?

Reply:
**Original :** XAML file: designed for advanced themes, styles change in runtime and can be shared between the applications

The exact meaning of the original sentence should be checked by @mihashco, but to my understanding, 
as [I commented](https://github.com/Samsung/tizen-docs/pull/1386#discussion_r560629916),
the meaning was that XAML is designed mostly for 3 purposes - "advanced themes" / "realtime change without rebuilding (styles change in runtime)" / "importable to other apps (can be shared between the applications)"

The 3rd part is also objects of the sentence as like "advanced themes", rather than additional description.

As you said, you left 1st object, and took out 2nd 3rd objects from the sentence to make a new sentence, considering them as continuation of description...

To me it sounds like changing the sentence like below.

Nawab reviews PR 111, 222, and 333 => Nawab reviews PR 111. 222 and 333.

"222 and 333." doesn't seem good as a sentence so I added "He" to the second part too, to maintain your suggestion.

Nawab reviews PR 111. He also do 222 and 333.

Reply:
@mijin85cho thanks, 
I got how you see the sentence. 

Its just that I see the sentences differently.

Ist part XAML files designed for advanced themes. ( 1 sentence).
Even if we implement XAML for non advanced themes, the other two will still work. 

Styles change at runtime.

Styles can be shared between applications. 

Both the things are good to go but lets go with your sentence